cookbook/integration/integration.py

```diff
@@ -65,45 +65,32 @@ def do_export(self, recipes):
         """
         Perform the export based on a list of recipes
         :param recipes: list of recipe objects
-        :return: HttpResponse with a ZIP file that is directly downloaded
+        :return: HttpResponse with the file of the requested export format that is directly downloaded (When that format involve multiple files they are zipped together) 
         """
 
-        # TODO this is temporary, find a better solution for different export formats when doing other exporters
-        if self.export_type != ImportExportBase.RECIPESAGE:
-            export_zip_stream = BytesIO()
-            export_zip_obj = ZipFile(export_zip_stream, 'w')
-
-            for r in recipes:
-                if r.internal and r.space == self.request.space:
-                    recipe_zip_stream = BytesIO()
-                    recipe_zip_obj = ZipFile(recipe_zip_stream, 'w')
-
-                    recipe_stream = StringIO()
-                    filename, data = self.get_file_from_recipe(r)
-                    recipe_stream.write(data)
-                    recipe_zip_obj.writestr(filename, recipe_stream.getvalue())
-                    recipe_stream.close()
-                    try:
-                        recipe_zip_obj.writestr(f'image{get_filetype(r.image.file.name)}', r.image.file.read())
-                    except ValueError:
-                        pass
-
-                    recipe_zip_obj.close()
-                    export_zip_obj.writestr(str(r.pk) + '.zip', recipe_zip_stream.getvalue())
-
-            export_zip_obj.close()
-
-            response = HttpResponse(export_zip_stream.getvalue(), content_type='application/force-download')
-            response['Content-Disposition'] = 'attachment; filename="export.zip"'
-            return response
+        files = self.get_files_from_recipes(recipes, self.request.COOKIES)
```

Comment:
i dont understand why the request cookie is passed, it isn't used anywhere or isn't it. I will check once merged and remove it if not used but maybe comment on this if it is not safe to remove the cookie parameter

Reply:
ah i see its not used for all but for some of the exporters